packages/editor/src/components/rich-text/index.js

```diff
@@ -987,26 +994,24 @@ const RichTextContainer = compose( [
 	withSafeTimeout,
 ] )( RichText );
 
-RichTextContainer.Content = ( { value, format, tagName: Tag, multiline, ...props } ) => {
+RichTextContainer.Content = ( { value, tagName: Tag, ...props } ) => {
 	let html = value;
 
 	// Handle deprecated `children` and `node` sources.
 	if ( Array.isArray( value ) ) {
 		html = children.toHTML( value );
-	} else if ( format !== 'string' ) {
-		html = toHTMLString( value, multiline );
 	}
 
 	const content = <RawHTML>{ html }</RawHTML>;
 
 	if ( Tag ) {
-		return <Tag { ...props }>{ content }</Tag>;
+		return <Tag { ...omit( props, [ 'format', 'multiline' ] ) }>{ content }</Tag>;
```

Comment:
Are these considered deprecated? Will they ever be removed?

Reply:
We can remove multiline as it was added after the last release.